tests/rptest/services/action_injector.py

```diff
@@ -222,9 +227,14 @@ def all_nodes_started(nodes):
         )
         wait_until(lambda: all_nodes_started(self.redpanda.started_nodes()),
                    timeout_sec=self.config.cluster_start_lead_time_sec,
-                   backoff_sec=2,
+                   backoff_sec=0.5,
                    err_msg='Cluster did not recover after failures')
 
+    def join(self, *args, **kwargs):
```

Comment:
It used to be recommended not to override `join` (or anything other than init and run really) in `threading.Thread`. I cannot find any latest references about it though. It seems like this should work okay.